src/store.js

```diff
@@ -15,6 +15,21 @@ var store = {
     auth: false
 };
 
+
+store.resetStore = function() {
+    // TODO : I'm thinking we need some user/trader objects that manage this.
+    this.user = {
+        id: null,
+        traders : []
+    };
+    this.trader = {
+        id : null,
+        pendedVouchers : []
+    };
+    this.vouchers = this.vouchers.splice(0, this.vouchers);
```

Comment:
Is this intended to clear the array? What is the value of this.vouchers at this point?